README.md

```diff
@@ -51,6 +51,7 @@ The Cloud Storage sink connector supports the following properties.
 | `sliceTopicPartitionPath` | Boolean| False |false | When it is set to `true`, split the partitioned topic name into separate folders in the bucket path. |
 | `withMetadata` | Boolean | False |false | Save message attributes to metadata. |
 | `withTopicPartitionNumber` | Boolean | False |true | When it is set to `true`, include topic partition number to the object path. |
+| `bytesFormatTypeSeparator` | String | False |"0x10" | Inserted between records for bytes formatType. Defaults to '0x10'. An input record that contains the line separator will look like multiple records in the output object. |
```

Comment:
Could we change it to :
It is inserted between records for the `formatType` of bytes. By default, it is set to '0x10'. An input record that contains the line separator looks like multiple records in the output object.

Reply:
thanks, addressed in the latest commit